terraform-plans/configs/charmed-openstack-exporter-snap_main.tfvars

```diff
@@ -0,0 +1,23 @@
+repository             = "charmed-openstack-exporter-snap"
+repository_description = "Snap package for the OpenStack exporter"
+branch                 = "main"
+workflow_files = {
+  codeowners = {
+    source      = "./templates/github/CODEOWNERS.tftpl"
+    destination = ".github/CODEOWNERS"
+    variables   = {}
+  }
+  promote = {
+    source      = "./templates/github/snap_promote.yaml.tftpl"
+    destination = ".github/workflows/promote.yaml"
+    variables   = {}
+  }
+  release = {
+    source      = "./templates/github/snap_release.yaml.tftpl"
+    destination = ".github/workflows/release.yaml"
+    variables = {
+      branch = "main"
+      include_check_job = false
```

Comment:
I don't think this will work; with the structure we have, terraform will enforce all values here to be the same type (cannot mix boolean and string).